src/components/SocialShare.vue

```diff
@@ -16,7 +16,7 @@ export default {
   props: {
     pageUrl: {
       type: String,
-      default: '',
+      required: true,
```

Comment:
This should be `default: ""` instead of required 
I recently fixed it as it was causing crashes